cmd/gokey/main.go

```diff
@@ -30,6 +31,7 @@ func init() {
 	flag.StringVar(&realm, "r", "", "password/key realm (most probably purpose of the password/key)")
 	flag.StringVar(&output, "o", "", "output path to store generated key/password (default stdout)")
 	flag.BoolVar(&unsafe, "u", false, "UNSAFE: allow key generation without a seed")
+	flag.IntVar(&passwordLength, "l", 10, `password length (only used for "pass" type)`)
```

Comment:
this probably needs some input validation - what if the user puts `0` or `-1` or `1234567890`?

Reply:
Just pushed some changes. Length now works on both passwords and raw symmetric keys. 

I wanted to keep the defaults for each output type for backward compatibility. So, `genRaw` checks if `length` was passed, if it wasn't then it sets length to `32`. Passwords still use the default of `10` characters.

It also does some input validation, but it may need to be tweaked.